src/main/java/taxiapp/jdbc/ManufacturerDaoJdbcImpl.java

```diff
@@ -0,0 +1,115 @@
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't create manufacturer", e);
+        }
+        return manufacturer;
+    }
+
+    @Override
+    public Optional<Manufacturer> get(Long manufacturerId) {
+        String query = "SELECT * FROM manufacturers WHERE id = ? AND deleted = false;";
+        try (Connection connection = ConnectionUtil.getConnection()) {
+            PreparedStatement statement = connection.prepareStatement(query);
+            statement.setLong(1, manufacturerId);
+            ResultSet resultSet = statement.executeQuery();
+            if (resultSet.next()) {
+                return Optional.of(parseResultSet(resultSet));
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't get manufacturer", e);
+        }
+        return Optional.empty();
+    }
+
+    @Override
+    public List<Manufacturer> getAllManufacturer() {
+        String query = "SELECT * FROM manufacturers WHERE deleted = false;";
+        List<Manufacturer> manufacturerList = new ArrayList<>();
+        try (Connection connection = ConnectionUtil.getConnection()) {
+            PreparedStatement statement = connection.prepareStatement(query);
+            ResultSet resultSet = statement.executeQuery();
+            while (resultSet.next()) {
+                manufacturerList.add(parseResultSet(resultSet));
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't get all manufacturers", e);
+        }
+        return manufacturerList;
+    }
+
+    @Override
+    public Manufacturer update(Manufacturer manufacturer) {
+        Manufacturer oldValue = get(manufacturer.getId()).orElseThrow();
+        String query = "UPDATE manufacturers SET name = ?,"
+                + "country = ? WHERE id = ? AND deleted = false;";
+        try (Connection connection = ConnectionUtil.getConnection()) {
+            PreparedStatement statement = connection.prepareStatement(query);
+            statement.setString(1, manufacturer.getName());
+            statement.setString(2, manufacturer.getCountry());
+            statement.setLong(3, manufacturer.getId());
+            if (statement.executeUpdate() != 1) {
+                throw new RuntimeException("update != 1");
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't update manufacturers", e);
+        }
+        return oldValue;
+    }
+
+    @Override
+    public boolean delete(Long manufacturerId) {
+        String query = "UPDATE manufacturers SET deleted = true WHERE id = ? AND deleted = false;";
+        try (Connection connection = ConnectionUtil.getConnection()) {
+            PreparedStatement statement = connection.prepareStatement(query);
+            statement.setLong(1, manufacturerId);
+            if (statement.executeUpdate() != 1) {
+                return false;
+            }
```

Comment:
exucuteUpdate возвращает количество измененных строк
```suggestion
            return statement.executeUpdate() > 0;
```